### PR TITLE
Fixes #3234 - Add an ability to prefill the form with postMessage

### LINKED ISSUE #3234: Add an ability to prefill the form with postMessage

### PR DESCRIPTION
**Pull request by ksy36**
r? @miketaylr

### CODE REVIEW

File: tests/functional/reporting-postMessage.js [hunk 1]
```diff
@@ -50,40 +50,3 @@ registerSuite("Reporting through postMessage", {
     },
   },
 });
-
-registerSuite("Reporting through postMessage for the wizard", {
-  tests: {
-    "postMessaged object to the new form"() {
```
Comment (ksy36) [inline tests/functional/reporting-postMessage.js:56]: (this was a duplicate of the suite above)

File: webcompat/static/js/lib/wizard/steps/hidden.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    src: {
+      element: $("#reported_with"),
+    },
+    extra_labels: {
+      element: $("#extra_labels"),
+      stringify: true,
+    },
+  };
+
+  Object.keys(config).forEach((key) => {
```
Comment (miketaylr) [inline webcompat/static/js/lib/wizard/steps/hidden.js:30]: This is cool 👍

**Comment by miketaylr** [comment]
(and nice that tests are passing now 👏 )

**Comment by ksy36** [comment]
Thanks!

**Pull request merged**